cg/meta/workflow/raredisease.py

```diff
@@ -1,25 +1,27 @@
 """Module for Raredisease Analysis API."""
 
 import logging
+from abc import ABC
 from pathlib import Path
 
 from cg.constants import GenePanelMasterList, Workflow
-from cg.constants.gene_panel import GENOME_BUILD_37
 from cg.constants.subject import PlinkPhenotypeStatus, PlinkSex
+from cg.constants.gene_panel import GENOME_BUILD_37
+from cg.constants.nf_analysis import RAREDISEASE_METRIC_CONDITIONS
 from cg.meta.workflow.analysis import add_gene_panel_combo
 from cg.meta.workflow.nf_analysis import NfAnalysisAPI
 from cg.models.cg_config import CGConfig
-from cg.models.nf_analysis import WorkflowParameters
 from cg.models.raredisease.raredisease import (
     RarediseaseSampleSheetEntry,
     RarediseaseSampleSheetHeaders,
 )
-from cg.store.models import CaseSample
+from cg.models.nf_analysis import WorkflowParameters
+from cg.store.models import CaseSample, Sample
 
 LOG = logging.getLogger(__name__)
 
 
-class RarediseaseAnalysisAPI(NfAnalysisAPI):
+class RarediseaseAnalysisAPI(NfAnalysisAPI, ABC):
```

Comment:
```suggestion
class RarediseaseAnalysisAPI(NfAnalysisAPI):
```
Why is this needed here?

Reply:
Removed 👍